.travis.yml

```diff
@@ -0,0 +1,17 @@
+language: python
+python:
+  - "3.3"
+  - "3.2"
+  - "2.7"
+  - "2.6"
+  - "2.5"
+  - "pypy"
```

Comment:
This PR is still a good idea but can we change the Python versions to 2.7, 3.4, 3.5, 3.6?  Also remove the two __—use-mirrors__ because pip no longer supports this option.

Reply:
I recommend forking and making a new PR for this. I would drop 3.4 too.